includes/wikia/DefaultSettings.php

```diff
@@ -1797,3 +1798,8 @@
  * flag for using (possibly external) service for managing a user's preferences
  */
 $wgPreferencesUseService = false;
+
+/**
+ * Parser Tag
+ */
+require_once "$IP/extensions/wikia/SoundCloudTag/SoundCloudTag.setup.php" ;
```

Comment:
@gabrys - we do have includes / requires in `wikia/DefaultSettings.php`, but they are used for core features - e.g. WikFactory, AssetsManager.

`SoundCloudTag` looks like a Yet Another Extension (TM) that can be enabled / disabled without changing the functionality of the whole site. I'd move the inclusion to the config repo.